### DISCUSSION TITLE
API "create new user" and "create a new conversation" functions

### DISCUSSION THREAD

**Discussion by Henriqueccm7** [langgenius/dify#4538] [Suggestion]
Please add a function in the API that I can create a new user and another one for start a new conversation, returning an new conversation_id

**Comment by patryk20120** [comment]
Why would you like to create users by API calls?

**Comment by luiszxcv** [comment]
@patryk20120 I don't understand how this feature doesn't exist yet, for example I have a lead who called me on WhatApp and I want to send the information by creating this lead and this conversation on dify to use an assistant, how do I proceed?

**Reply by crazywoola** [reply]
If you send requests to /chat-messages, it will return the `conversation_id` in it's response. And the user_id is defined by yourself regarding to your own business logic.

**Comment by mandos21** [comment]
This would be useful for bootstrapping a system that is spun up with a set of required users without having to perform these additions through the UI.

**Reply by phpia** [reply]
Absolutly
